lib/reek/cli/report/report.rb

```diff
@@ -102,6 +103,15 @@ def show
       end
 
       #
+      # Displays a list of smells in JSON format
+      # JSON with empty array for 0 smells
+      class JsonReport < Base
```

Comment:
Does it have to be `JsonReport` rather than `JSONReport`?

(The only reason for these capitalisation abominations I heard about was related to Rails’ code autoloading, but I’m working on a Rails 2.3 app at the moment and even it would find `JSONRerport` in `json_report.rb` just fine.)